### ISSUE TITLE
Scrollbar aria-orientation required property?

### ISSUE THREAD

**Issue by iannightingale** [w3c/aria-practices#705]
[https://www.w3.org/TR/wai-aria-1.1/#scrollbar](https://www.w3.org/TR/wai-aria-1.1/#scrollbar)

The Scrollbar section states:

> Elements with the role scrollbar have an implicit aria-orientation value of vertical.

However, `aria-orientation` is listed in the "Required States and Properties:" row of the Scrollbar characteristic table, suggesting that it is required and doesn't have an implicit value? These seem incompatible.

**Comment by jnurthen** [comment]
Can you please log this against the ARIA specification.
https://github.com/w3c/aria/issues

**Comment by iannightingale** [comment]
https://github.com/w3c/aria/issues/780

Sorry, thanks!
